draft-ietf-bfcpbis-rfc4583bis.xml

```diff
@@ -488,8 +489,6 @@
         <t>
           Note: If the offerer includes an SDP 'floorctrl' attribute with a 'c-s' attribute value, or both a 'c-only' and a 's-only' attribute value, in the offer, 
```

Comment:
"c-s" should be removed from the "In addition..." line above this section, and possible also from this line.

Although we specify an _answerer_ can parse "c-s", we should not be permitting a spec-compliant offerer to send "c-s". At least, that's the approach I had assumed we should be taking.

Reply:
Will fix as suggested.